### DISCUSSION TITLE
Mouse Wheel Scroll moves Pivot away when using Skia.GTK

### DISCUSSION THREAD

**Discussion by NicoAICP** [unoplatform/uno#9032] [Q&A]
Hello There.

If i use a Pivot item from the WinUI 2 stuff, i always get the issue in Skia.GTK (not in UWP) that if i scroll while having a pivot element close to me, it scrolls away the pivot element (as if i had an big webpage that i could scroll left and right, and i scroll away from the pivot)
Is there any way to fix this?
I can provide an gif for this issue if needed

**Comment by jeromelaban** [comment]
@dr1rrb

**Comment by NicoAICP** [comment]
Has there been any known news to this?
I tried a few things, i still havent gotten the Issue away. For now it isnt that much of an issue for me, but if i plan to release it, it would be a deal breaker, if the whole ui disappears because you accidently hit the scroll wheel.

**Reply by jeromelaban** [reply]
There are not updates on this issue, but let's track it here: https://github.com/unoplatform/uno/issues/9146.

If you need more information about timelines, let us know by sending us an email at info@platform.uno.
